src/libraries/Directory.Build.targets

```diff
@@ -49,7 +49,7 @@
       <RuntimePath>$(RuntimePath)</RuntimePath>
     </BinPlaceConfiguration>
     <!-- binplace to directories for packages -->
-    <BinPlaceConfiguration Condition="'$(IsNETCoreApp)' == 'true' and '$(BuildingNETCoreAppVertical)' == 'true'" Include="netcoreapp-$(_bc_OSGroup)">
+    <BinPlaceConfiguration Condition="'$(IsNETCoreApp)' == 'true' and '$(BuildingNETCoreAppVertical)' == 'true'" Include="netcoreapp5.0-$(_bc_OSGroup)">
```

Comment:
Here and in many other places you are hardcoding netcoreapp5.0. This is very unfortunate as we will bump the tfm at least every year going forward. Can you please update those instances as well to use the property?